src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -0,0 +1,256 @@
+                <View style={[styles.flex1, isSmallScreenWidth ? styles.workspaceSectionMobile : styles.workspaceSection]}>
+                    <Section
+                        title={translate('workspace.accounting.title')}
+                        subtitle={translate('workspace.accounting.subtitle')}
+                        isCentralPane
+                        subtitleMuted
+                        titleStyles={styles.accountSettingsSectionTitle}
+                        childrenStyles={styles.pt5}
+                    >
+                        {policyIsConnectedToAccountingSystem && (
+                            <View
+                                ref={threeDotsMenuContainerRef}
+                                style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}
+                            >
+                                <MenuItem
+                                    title={translate('workspace.accounting.qbo')}
+                                    description={translate(isSyncInProgress ? 'workspace.accounting.importing' : 'workspace.accounting.lastSync')}
+                                    icon={Expensicons.QBORound}
+                                    iconHeight={variables.avatarSizeNormal}
+                                    iconWidth={variables.avatarSizeNormal}
+                                    wrapperStyle={styles.sectionMenuItemTopDescription}
+                                    interactive={false}
+                                />
+                                {isSyncInProgress ? (
+                                    <ActivityIndicator
+                                        style={[styles.popoverMenuIcon]}
+                                        color={theme.spinner}
+                                    />
+                                ) : (
+                                    <ThreeDotsMenu
+                                        onIconPress={() => {
+                                            threeDotsMenuContainerRef.current?.measureInWindow((x, y, width, height) => {
+                                                setThreeDotsMenuPosition({
+                                                    horizontal: x + width,
+                                                    vertical: y + height,
+                                                });
+                                            });
+                                        }}
+                                        menuItems={threeDotsMenuItems}
+                                        anchorPosition={threeDotsMenuPosition}
+                                        anchorAlignment={{horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT, vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.TOP}}
+                                        shouldOverlay
+                                    />
+                                )}
+                            </View>
+                        )}
```

Comment:
This feels like a disconnected menu item. Can we add this to `qboConnectionMenuItems`?

Reply:
not really sure that we should add this one item to qboConnectionMenuItems
It has too many specific logic - like activity indicator, tree dots, changes during isSyncInProgress

Reply:
It only has 2 more components that are rendered in the same row as any other menu item component e.g. the brick  road indicator. We can just extend MenuItem to take 2 more props.

Reply:
Hi @s77rt I initially tried it that way. The `ThreeDotsMenu` wouldn't stay in position if rendered within the `MenuItem` itself, reason why we had to keep it this way.

Reply:
Ah I see. It seems we can just add a wrapping View with styles `styles.flex1, styles.flexRow` and attach the ref to this View


<img width="641" alt="Screenshot 2024-03-22 at 1 35 56 AM" src="https://github.com/Expensify/App/assets/16493223/bd284e8f-bc7a-47c1-a3fb-bdd56ac4854f">

Reply:
> Ah I see. It seems we can just add a wrapping View with styles `styles.flex1, styles.flexRow` and attach the ref to this View
> 
> <img alt="Screenshot 2024-03-22 at 1 35 56 AM" width="641" src="https://private-user-images.githubusercontent.com/16493223/315843325-bd284e8f-bc7a-47c1-a3fb-bdd56ac4854f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExMDA4MTksIm5iZiI6MTcxMTEwMDUxOSwicGF0aCI6Ii8xNjQ5MzIyMy8zMTU4NDMzMjUtYmQyODRlOGYtYmM3YS00N2MxLWEzZmItYmRkNTZhYzQ4NTRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDA5NDE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1NTIyMDM4NTQxNTczMWIwZTE5NzY2ODZmNTRmYzBjZTc2ZjJiZjkyNTkwZGI3NmE4NmNlOGM0ODhkMzgzYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.n5ngVhjm9Y8Ug1vkRzxj5XcBUbkPf1GvoM11x3mwHBA">

@s77rt for the hint on this, i guess listing the ref in useMemo dep tray was enough too 😉